File: app/src/main/java/com/dkellycollins/triggerfinger/managers/state/IStateManager.java
package com.dkellycollins.triggerfinger.managers.state;

/**
 * Represents a manager that manages the state of the game.
 */
public interface IStateManager {

    /**
     * Called during the update phase.
     * @param deltaTime The time since the last update.
     */
    void Update(int deltaTime);
}

File: src/apps/Android-Kotlin/app/src/main/kotlin/com/dkellycollins/triggerfinger/managers/state/IStateManager.kt
package com.dkellycollins.triggerfinger.managers.state

/**
 * Represents a manager that manages the state of the game.
 */
interface IStateManager {

    /**

     */
    fun init()

    /**
     * Called during the update phase.
     * @param deltaTime The time in milliseconds since the last update.
     */
    fun update(deltaTime: Long)
}
